bratislava/olo.sk
# #380 Use new `color` field from Service Categories instead of hardcoded colors

- use new `color` field instead of hardcoded variants

### TODOs in next PR:
- cleanup unused code - remove old field and hardcoded colors

### How to test:

When connecting yous Next to production Strapi, new colors are already set and should work as expected.

OR in your local Strapi

- set `color` for you Services Categories
- update Services meilisearch index
![image](https://github.com/user-attachments/assets/e2d2084c-e595-492a-9c63-99fcd0c6e19b)
![image](https://github.com/user-attachments/assets/2fe2c41c-9951-4c1b-9439-601aaf9e0d10)
